src/service/cli/server.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+
+const http = require(`http`);
+const fs = require(`fs`).promises;
+const chalk = require(`chalk`);
+
+const DEFAULT_PORT = 3000;
+const FILENAME = `mocks.json`;
+const HttpCode = {
+  OK: 200,
+  NOT_FOUND: 404,
+  INTERNAL_SERVER_ERROR: 500,
+  FORBIDDEN: 403,
+  UNAUTHORIZED: 401,
+};
+
+const sendResponse = (res, statusCode, message) => {
+  const template = `
+    <!Doctype html>
+      <html lang="ru">
+      <head>
+        <title>With love from Node</title>
+      </head>
+      <body>${message}</body>
+    </html>`.trim();
+
+  res.writeHead(statusCode, {
+    'Content-Type': `text/html; charset=UTF-8`,
+  });
+
+  res.end(template);
+};
+
+const onClientConnect = async (req, res) => {
+  const notFoundMessageText = `Not found`;
+
+  switch (req.url) {
+    case `/`:
+      try {
+        const fileContent = await fs.readFile(FILENAME);
+        const mocks = JSON.parse(fileContent);
+        const message = mocks.map((post) => `<li>${post.title}</li>`).join(``);
+        sendResponse(res, HttpCode.OK, `<ul>${message}</ul>`);
+      } catch (err) {
+        sendResponse(res, HttpCode.NOT_FOUND, notFoundMessageText);
+      }
+      break;
+    default:
+      sendResponse(res, HttpCode.NOT_FOUND, notFoundMessageText);
+      break;
```

Comment:
```suggestion
    case `/`: {
      try {
        const fileContent = await fs.readFile(FILENAME);
        const mocks = JSON.parse(fileContent);
        const message = mocks.map((post) => `<li>${post.title}</li>`).join(``);
        sendResponse(res, HttpCode.OK, `<ul>${message}</ul>`);
      } catch (err) {
        sendResponse(res, HttpCode.NOT_FOUND, notFoundMessageText);
      }
      break;
     }
    default: {
      sendResponse(res, HttpCode.NOT_FOUND, notFoundMessageText);
      break;
     }
```
не забывай о скобках